src/previewSupport/handleConsoleLogs.js

```diff
@@ -0,0 +1,32 @@
+import channel from './channel';
+
+function notifyChannel(args) {
+  if (args.length > 0) {
+    const output = args.map(arg => JSON.stringify(arg)).join(' ');
+    channel.notify({
+      method: 'log',
+      params: {output},
+    });
+  }
+}
+
+export default function handleConsoleLogs() {
+  const consoleFunctions = ['log', 'info', 'warn', 'error'];
+
+  // eslint-disable-next-line no-console
+  if (console.debug) {
+    consoleFunctions.push('debug');
+  }
```

Comment:
So I actually think we do want to allow `console.debug` even if the browser doesn’t define it—should just do a noop for the `browserFunction`. WDYT?